src/main/java/mat/server/service/FhirValidationReportService.java

```diff
@@ -28,19 +19,29 @@
 
 import freemarker.template.Configuration;
 import freemarker.template.TemplateException;
+import gov.cms.mat.fhir.rest.dto.ConversionOutcome;
+import gov.cms.mat.fhir.rest.dto.ConversionResultDto;
+import gov.cms.mat.fhir.rest.dto.CqlConversionError;
+import gov.cms.mat.fhir.rest.dto.CqlConversionResult;
+import gov.cms.mat.fhir.rest.dto.FhirValidationResult;
+import gov.cms.mat.fhir.rest.dto.LibraryConversionResults;
+import gov.cms.mat.fhir.rest.dto.MatCqlConversionException;
+import gov.cms.mat.fhir.rest.dto.ValueSetConversionResults;
 import mat.client.shared.MatRuntimeException;
 import mat.dao.clause.MeasureDAO;
 import mat.model.clause.Measure;
+import mat.model.clause.ModelTypeHelper;
 import mat.shared.DateUtility;
 
-@Service("fhirValidationService")
+@Service
 public class FhirValidationReportService {
 
+    private static final Log logger = LogFactory.getLog(FhirValidationReportService.class);
     private static final String DATE_FORMAT = "dd-MMM-YYYY";
     private static final String TIME_FORMAT = "hh:mm aa";
     private static final String NO_MEASURE_FOUND_ERROR = "The measure with that measure id does not exist.";
-    private static final String CONVERSION_SERVICE_ERROR = "An error occurred while validating the FHIR conversion. Please try again later.";
-    private static final Log logger = LogFactory.getLog(FhirValidationReportService.class);
+    private static final String CONVERSION_SERVICE_ERROR = "An error occurred while validating the FHIR conversion. Please try again later. If this continues please contact the MAT help desk.";
+    private static final String REPORT_FTL = "fhirvalidationreport/fhir_validation_report.ftl";
```

Comment:
For consistency, could you move this to the properties file, please?

Reply:
Doesn't make sense to me to move that to properties file. Those values are not configurable.

Reply:
or, what do you mean?

Reply:
Trying to avoid having hard coded template paths in multiple files. The email templates are currently set in the MAT.properties, but they're the only ones.

Reply:
> Trying to avoid having hard coded template paths in multiple files.
Why? It doesn't make sense to me. We are not going to override that. No reason to make things more complex than needed.